cdap-app-fabric/src/main/java/io/cdap/cdap/app/guice/AppFabricServiceRuntimeModule.java

```diff
@@ -330,6 +331,7 @@ protected void configure() {
       });
       bind(ProfileService.class).in(Scopes.SINGLETON);
       bind(PreferencesFetcher.class).to(LocalPreferencesFetcherInternal.class).in(Scopes.SINGLETON);
+      bind(RemoteTaskExecutor.class).in(Scopes.SINGLETON);
```

Comment:
This class seems like a simple helper. You don't need to use guice for it. Just `new` and use it.

Reply:
Done